roles/aws/aws_admin_tools/tasks/create.yml

```diff
@@ -0,0 +1,74 @@
+- name: Create stage on API gateway
+  ansible.builtin.command: >-
+    aws apigateway create-stage
+    --rest-api-id "{{ _api_gate.id }}"
+    --stage-name "prod"
+    --deployment-id "{{ _main_api_deploy.id }}"
+    --region "{{ _aws_region }}"
+  register: _main_api_stage
+  when: _api_index | length == 0
+
+- name: Create resources and set methods on API Gateway.
+  ansible.builtin.include_tasks: create_methods.yml
+  loop: "{{ aws_admin_tools.functions }}"
+
+- name: Obtain all information for a single WAF
```

Comment:
Same, missing full stop.

Reply:
Actually, lots of missing full stops. Let's make sure all task names start with a capital letter and end in a full stop. For tidiness. :slightly_smiling_face: